### PR TITLE
Mm 43665 - modify the learn more trial modal

### PR DESCRIPTION
**Pull request by pvev**
#### Summary
This PR adds the design changes to the learn more about trial modal

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-43665

#### Related Pull Requests
n/a

#### Screenshots

Captura de pantalla 2022-06-03 a las 15 48 45
Captura de pantalla 2022-06-03 a las 15 48 31

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: components/learn_more_trial_modal/learn_more_trial_modal.tsx [hunk 1]
```diff
@@ -47,60 +96,53 @@ const LearnMoreTrialModal = (
...
+            pageURL: 'https://docs.mattermost.com/onboard/sso-saml.html',
+            buttonLabel,
         },
         {
-            id: 'complianceExport',
-            title: formatMessage({id: 'trial_benefits.modal.complianceExportTitle', defaultMessage: 'Automate compliance exports'}),
-            description: formatMessage({id: 'trial_benefits.modal.complianceExportDescription', defaultMessage: 'Run daily compliance reports and export them to a variety of formats consumable by third-party integration tools such as Smarsh (Actiance).'}),
+            id: 'ldap',
+            title: formatMessage({id: 'learn_more_about_trial.modal.ldapTitle', defaultMessage: 'Synchronize your Active Directory/LDAP groups'}),
+            description: formatMessage({id: 'learn_more_about_trial.modal.ldapDescription', defaultMessage: 'Run daily compliance reports and export them to a variety of formats consumable by third-party integration tools such as Smarsh (Actiance).'}),
```
Comment (pvev) [inline components/learn_more_trial_modal/learn_more_trial_modal.tsx:119]: This text description needs to be updated by UX

**Review by JulienTant — APPROVED** [review]
:shipit:

File: components/learn_more_trial_modal/learn_more_trial_modal.tsx (hunk 1 above)
Comment (pvev) [reply components/learn_more_trial_modal/learn_more_trial_modal.tsx]: updated!

File: i18n/en.json [hunk 2]
```diff
@@ -3672,6 +3672,13 @@
   "last_users_message.others": "{numOthers} others ",
   "last_users_message.removed_from_channel.type": "were **removed from the channel**.",
   "last_users_message.removed_from_team.type": "were **removed from the team**.",
+  "learn_more_about_trial.modal.ldapDescription": "Use AD/LDAP groups to organize and apply actions to multiple users at once. Manage team and channel memberships, persmissions and more.",
```
Comment (julmondragon) [inline i18n/en.json:3675]: `permissions`

File: i18n/en.json [hunk 3]
```diff
@@ -3672,6 +3672,13 @@
...
+  "learn_more_about_trial.modal.ldapTitle": "Synchronize your Active Directory/LDAP groups",
+  "learn_more_about_trial.modal.systemConsoleDescription": "Use System Roles to give designated users read and/or write access to select sections of System Console.",
+  "learn_more_about_trial.modal.systemConsoleTitle": "Provide controlled access to the System Console",
+  "learn_more_about_trial.modal.useSsoDescription": "Sign on quickly and easily with our SSO feature that works with OpenID, SAML, Google, and 0365.",
+  "learn_more_about_trial.modal.useSsoTitle": "Use SSO (with OpenID, SAML, Google, 0365)",
+  "learn_more_trial_modal_step.learnMoreAboutFeature": "Learn more about this feature.",
   "learn_more_trial_modal.pretitle": "With Enterprise, you can....",
```
Comment (julmondragon) [inline i18n/en.json:3682]: Four dots. Maybe better to use the ellipsis char `…`

**Review by julmondragon — APPROVED** [review]
Looks good!

File: components/common/carousel/carousel_button.tsx [hunk 4]
```diff
@@ -31,6 +38,18 @@ const CarouselButton: React.FC<Props> = ({direction, moveSlide, disabled}: Props
         formatMessage({id: 'carousel.PreviousButton', defaultMessage: 'Previous'});
 
     const disabledClass = disabled ? ' disabled' : '';
+    const orientation = direction === Destination.NEXT ? 'right' : 'left';
+
+    if (btnsStyle === BtnStyle.CHEVRON) {
+        return (
+            <div
+                className={`chevron-button chevron-${orientation} ${direction}-btn` + disabledClass}
```
[the remark this answers is not in the document]
Comment (pvev) [reply components/common/carousel/carousel_button.tsx:46]: orientation is `left/right `and direction is `prev/next.` The thing is that for the compass design icon we need the orientation that will depend on the direction. And the button text will depend also on the direction `Next/Previous`

File: i18n/en.json (hunk 3 above)
Comment (pvev) [reply i18n/en.json:3682]: nice catch!

File: i18n/en.json (hunk 2 above)
Comment (pvev) [reply i18n/en.json:3675]: nice catch!

**Comment by pvev** [comment]
@stevemudie e2e seemed to pass successfully https://mattermost-cypress-report.s3.amazonaws.com/202603-ce47b0f-onprem-ent-webapp-pr-10468-mattermost/mm-ee-testce47b0f/mochawesome.html 🎆

**Review by stevemudie — APPROVED** [review]
Looks great @pvev!

**Pull request merged**